package.json

```diff
@@ -35,27 +36,58 @@
     "fix": "npm run fix:lint && npm run fix:prettier",
     "start": "npm run develop",
     "serve": "gatsby serve",
-    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
-    "commit": "git-cz"
+    "commit": "git-cz",
+    "cypress:open": "cross-env TZ=UTC node_modules/.bin/cypress open",
+    "cypress:run": "cross-env TZ=UTC node_modules/.bin/cypress run",
+    "test": "cross-env NODE_ENV=test npm run test:unit && npm run test:e2e",
+    "test:unit": "nyc --silent --no-clean jest",
+    "test:e2e": "nyc --silent --no-clean start-server-and-test develop http://localhost:8000 cypress:run",
+    "pretest": "chmod +x scripts/**.* && cross-env ./scripts/pre-test.sh",
```

Comment:
We shouldn't need to always modify the scripts, once we change the permissions on them on your system, the permissions will be the same on everyones. So no need for `chmod +x scripts/**.*`

Reply:
I understand that and this is just a fail-safe.

Reply:
There technically won't be a situation where a file will lose its permissions. So I think it's safe to remove it since it's just doing the same thing over and over again.

Reply:
Okay, Boss. Will do. 
The fail-safe was from my side but this's off-topic so I won't include in this conversation.